pkgs/development/ocaml-modules/reason-react/ppx.nix

```diff
@@ -0,0 +1,35 @@
+{
+  buildDunePackage,
+  fetchurl,
+  lib,
+  ppxlib,
+}:
+
+let
+  version = "0.15.0";
+  minimalOCamlVersion = "4.14";
+in
+buildDunePackage {
+  pname = "reason-react-ppx";
+  inherit version;
+  inherit minimalOCamlVersion;
+  src = fetchurl {
+    url = "https://github.com/reasonml/reason-react/releases/download/${version}/reason-react-${version}.tbz";
+    hash = "sha256-+pPJo/b50vp4pAC/ygI1LHB5O0pDJ1xpcQZOdFP8Q80=";
+  };
+  buildInputs = [
+    ppxlib
+  ];
+  doCheck = false; # Needs to run in reason-react, see default.nix
+  meta = {
+    description = "React.js JSX PPX";
+    homepage = "https://github.com/reasonml/reason-react";
+    license = lib.licenses.mit;
+    maintainers = [
+      lib.maintainers.vog
+    ];
+  };
+}
+// {
+  inherit minimalOCamlVersion;
+}
```

Comment:
This is wrong. If you really want this attribute to be passed around, you must wrap it in `passthru`.

Reply:
I assume this should be tied to the melange version so maybe add it there and use it to passthru

Reply:
@vbgl @ulrikstrid Neither is needed (see my comment above), so I'll just leave that out.